tests/unittest/disaggregated/test_chunked_transfer.py

```
@@ -0,0 +1,1516 @@
+def test_pipelined_transfer_requires_chunked_prefill():
+    """ValueError when pipelined transfer is enabled without chunked prefill."""
+    from tensorrt_llm._torch.pyexecutor.kv_cache_transceiver import create_kv_cache_transceiver
+
+    cache_transceiver_config = CacheTransceiverConfig(
+        backend="NIXL",
+        enable_pipelined_transfer=True,
+    )
+
+    with pytest.raises(
+        ValueError,
+        match="enable_chunked_prefill is required when enable_pipelined_transfer is set.",
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)